java/src/main/java/ai/rapids/cudf/PartitionedTable.java

```diff
@@ -38,6 +38,10 @@
     this.partitionsOffsets = partitionOffsets;
   }
 
+  Table getTable() {
```

Comment:
We should probably make this public.  There is no harm in exposing it, and I could see others wanting to potentially get access to it.